autosar/package.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, name, parent=None, role=None):
         self.parent=parent
         self.role=role
         self.map={'elements':{}, 'packages':{}}
+        self.UnhandledParser = [] #[PackageParser] unhandled

```

Comment:
1. This should be a set from the very beginning.
2. Use a lower-case letter as first character.
self.unhandledParser = set()